src/conf.c

```diff
@@ -500,6 +535,16 @@ int ConfGetBool(const char *name, int *val)
  */
 int ConfGetChildValueBool(const ConfNode *base, const char *name, int *val)
 {
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
Well, I was cc-ing you to ask you if it will be compatible with the conf+rust changes.

We can have `#cfg[fuzz]` or something like that (as a way to `#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`)

But will still SMTP use `ConfGetChildValueBool` ? (just its implementation will be in rust)

Reply:
> But will still SMTP use `ConfGetChildValueBool` ? (just its implementation will be in rust)

For the most part.  Rust code should be able to use `serde` to convert raw config to a struct, but that won't work so well for the C.

Reply:
I probably wouldn't wait, but might need to come back to you for reworking when the time comes.